### ISSUE TITLE
Add the full functionality to "take a turn"

### ISSUE THREAD

**Issue by bromeostasis** [mwbennett/emojiphone#7]
Once #2 is complete, we'll need to set up the full functionality to take a turn in the game!

This includes:

* Sending a prompt to the correct user
* Receiving messages outside of the setup phase
* Associating an incoming message with a user (`SELECT * FROM turns WHERE turns.user.phone_number == input_phone_number AND isCurrent=TRUE`)
* Responding to incoming message upon successful receipt
  * Or sending an error if it's not their turn!
* Trigger the next turn or do nothing if the game is over (handled in #8) 

**Tasks**

Moved from #2

- [x] Accept Turn in initiateTurnConversation
- [x] Prompt next user to take turn after their turn is complete
- [ ] ~~Prompt user initially using turn~~ moving to #6 
- [ ] ~~Listen for all messages~~
- [ ] ~~Determine whether current message is coming from a user whose turn it is ☝️~~ 
- [x] Fill variable parts of turn conversation from db
- [x] Mark turn as complete in database (store message)
- [x] Set next user's turn to true in database
- [x] Initiate next turn after turn is complete
- [x] If last user's turn, do nothing (placeholder for #9 )
- [ ] ~~If it's easy, just handle #6 by calling method to setup turn conversation~~

**Dependencies**

#2 

**Out of scope**

* Text/emoji validation (handled in #3)
